projects/output-and-error-handling/parseRetryValue_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestParseRetryValue(t *testing.T) {
+
+	t.Run("parses valid numbers", func(t *testing.T) {
+
+		gotValue, gotErr := ParseRetryValue("5")
+		expectedValue := 5
+
+		if gotValue != expectedValue {
+			t.Errorf("expected %v but got %v", expectedValue, gotValue)
+		}
+
+		if gotErr != nil {
+			t.Errorf("expected %v as error but got %v", nil, gotErr)
+		}
+	})
+
+	t.Run("parses invalid numbers", func(t *testing.T) {
+
+		gotValue, gotErr := ParseRetryValue("a while")
+		expectedValue := 0
+		expectedErr := "couldn't parse header as int or timestamp"
+
+		if gotValue != expectedValue {
+			t.Errorf("expected %v but got %v", expectedValue, gotValue)
+		}
+
+		if !strings.Contains(gotErr.Error(), expectedErr) {
+			t.Errorf("expected %v to be contained in %v", expectedErr, gotErr)
+		}
+	})
+
+	t.Run("parses http time format", func(t *testing.T) {
+		duration := 5
+		httpTime := time.Now().Add(time.Duration(duration) * time.Second).Format(time.RFC1123)
+		expected := duration - 1
```

Comment:
There is a risk that this test may sometimes fail if you get unlucky and your computer happens to be running a bit slowly when this test is running... Can you think of an idea of how to write this test _without_ that risk?